examples/gno.land/r/demo/auction/auction.gno

```diff
@@ -0,0 +1,163 @@
+package auction
+
+import (
+	"bytes"
+	"std"
+	"strconv"
+	"time"
+
+	auctionp "gno.land/p/demo/auction"
+	"gno.land/p/demo/mux"
+)
+
+var (
+	auctionList []*auctionp.Auction
+	currentTime time.Time
+	// router      *mux.Router
+)
+
+var router = mux.NewRouter()
+
+func init() {
+	router.HandleFunc("", renderHomepage)
+	router.HandleFunc("create/", renderCreateAuction)
+	router.HandleFunc("upcoming/", renderUpcomingAuctions)
+	router.HandleFunc("ongoing/", renderOngoingAuctions)
+	router.HandleFunc("/auction/{id}", renderAuctionDetails)
```

Comment:
isn't this weird?

Reply:
I removed the auction state [22285536](https://github.com/gnolang/gno/pull/2265/commits/22285536cd244bb658d334e2ab87ae13d2c8828d)